client/src/api/.openapi-generator/FILES

```diff
@@ -1,8 +1,8 @@
-.gitignore
-.npmignore
-api.ts
-base.ts
-common.ts
-configuration.ts
-git_push.sh
-index.ts
+.gitignore
+.npmignore
+api.ts
+base.ts
+common.ts
+configuration.ts
+git_push.sh
+index.ts

```

Comment:
What changed in this file?

Reply:
> What changed in this file?

I believe EOL, I should revert it.
Thank you

Reply:
reverted